metadata-ingestion/tests/integration/snowflake/snowflake_golden.json

```diff
@@ -294,17 +283,6 @@
                     },
                     "nativeDataType": "VARCHAR(255)",
                     "recursive": false,
-                    "glossaryTerms": {
```

Comment:
Not sure why these are coming up on my local machine, but not failing on CI... @mayurinehate any clue here? This started failing locally since ac06cf3d3f33798d16258aa6553e292ac7e92d54

Reply:
Can you try creating fresh venv once (or use acryl-datahub-classify==0.0.8)? If that doesn't work, this needs to be fixed in code, maybe by increasing number of sample values generated in `mock_sample_values.return_value` function.

Reply:
Nice catch. Didn't realize classify code was in a separate package